apollo-composition/src/lib.rs

```diff
@@ -227,17 +224,21 @@ pub trait HybridComposition {
             subgraphs: connected_subgraphs,
             parsed_subgraphs,
             hints: connector_hints,
-        } = validate_connector_subgraphs(validated_subgraphs)?;
-        let override_errors = validate_overrides(parsed_subgraphs);
-        if !override_errors.is_empty() {
-            return Err(override_errors);
-        }
+        } = validate_connector_subgraphs(upgraded_subgraphs)?;
 
         // merge
         let merge_result = self
             .experimental_merge_subgraphs(connected_subgraphs)
             .await?;
 
+        // Extra connectors validation after merging.
```

Comment:
Moved this step after `merge` step above to mirror the current production behavior.